topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -0,0 +1,526 @@
+>    - {% icon param-file %} *"Specify an input file"*: `output_tabular` (output from **Remove Columns** {% icon tool %})
+>    - *"File has header?"*: `Yes`
+>    - *"FDR (adjusted P value) column number"*: `c4`
+>    - *"P value (raw) column number"*: `c3`
+>    - *"Log Fold Change column number"*: `c2`
+>    - *"Labels column number"*: `c1`
+>    - *"LogFC threshold to colour"*: `0.58`
+>    - *"Points to label"*: `Significant`
+>        - *"Only label top most significant"*: `40`
+>    - In *"Plot Options"*:
+>        - *"Plot title"*: `Differential Expression Volcano Plot`
+>
+>    > <tip-title> What does the Volcano Plot show? </tip-title>
+>    >
+>    > The Volcano Plot highlights genes with statistical significance (low P-values) and large fold changes. Genes meeting the significance thresholds are typically colored and labeled for easier identification. 
+>    {: .tip}
+>
+{: .hands_on}
+
+## What are the results of the Volcano Plot?
+
+Let's take a moment to interpret the Volcano Plot:
+![Volcano Plot](../../images/pseudobulk-analysis/VolcanoPlot.png)
+- Which genes are most significant?
+- How does the fold change correlate with the P-values?
+- Are there any unexpected patterns?
+
+> <question-title> Reflect on the Volcano Plot </question-title>
+>
+> 1. What is the significance of genes located at the extremes of the plot (e.g., high LogFC and low P-value)? 
+> 2. How many genes meet the significance threshold in this analysis?
+>
+> > <solution-title> Solution </solution-title>
+> >
+> > 1. Genes at the extremes of the Volcano plot are highly significant and exhibit strong differential expression. Consider exploring the biological significance of these genes to gain deeper insights into their roles.
+> > 2. In our current analysis, we identified only one downregulated gene, MTND1P23, with a logFC of -3.1956, in normal conditions compared to COVID-19 patients.
+> >
+> {: .solution}
+>
+{: .question}
+
+# Subsetting Samples from the Original AnnData Object
+
+In our previous analysis, we found that only one gene, **MTND1P23**, was identified as downregulated in our contrast. This result was obtained by "bulking" **all** cell types from our dataset and performing a differential expression analysis comparing normal vs. COVID-19 samples.
+
+Now, what if we refine our approach? For instance, instead of analysing all cell types together, what happens if we focus on a specific cluster, such as **T cells**, and perform the same comparison: normal vs. COVID-19? Would the results remain the same, or would this approach reveal additional insights?
+
+## Extracting observations of interest as AnnData object 
+
+> <hands-on-title> Use Manipulate AnnData Tools to extract observations </hands-on-title>
+>
+> 1. Use the {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.10.9+galaxy0) %} tool with the following parameters:
```

Comment:
In the latest version, this function does not exist anymore. We moved this function to Scanpy Filter tool. Please adjust this hands-on step accordingly and also update the workflow.